_damon_result.py

```diff
@@ -341,12 +355,8 @@ def aggregate_snapshots(snapshots):
 
 record_requests = {}
 def start_monitoring_record(file_path, file_format, file_permission):
-    try:
-        subprocess.check_output(['which', PERF])
-    except:
-        return None, 'perf is not installed'
     pipe = subprocess.Popen(
-            [PERF, 'record', '-a', '-e', 'damon:damon_aggregated', '-o',
+            [PERF, 'record', '-a', '-e', PERF_EVENT, '-o',
                 file_path])
     record_requests[pipe] = [file_path, file_format, file_permission]
     return pipe, None
```

Comment:
It looks there's no need to return error code here now. I will remove it and its callsite, then make it return `pipe` only in the next update.

Reply:
Thanks. I've just uploaded the follow up patch at #40.